api/internal/handler/route_online_debug/route_online_debug.go

```diff
@@ -49,6 +49,7 @@ var protocolMap map[string]ProtocolSupport
 func init() {
 	protocolMap = make(map[string]ProtocolSupport)
 	protocolMap["http"] = &HTTPProtocolSupport{}
+	protocolMap["https"] = &HTTPProtocolSupport{}
 }
```

Comment:
oh, we don't need a init function here, just a map is ok

Reply:
you could refer to:
https://github.com/uber-go/guide/blob/master/style.md#avoid-init

Reply:
> you could refer to:
> https://github.com/uber-go/guide/blob/master/style.md#avoid-init

thanks, updating~